### ISSUE TITLE
Discord power hungry mods

### ISSUE THREAD

**Issue by Jufinda** [keldaanCommunity/pokemonAutoChess#2033]
Game was lagging to the point it couldnt be played

went to discord and posted "game is literally unplayable"

insta kicked and banned.

had been in the discord for about 10 minutes since joining to see if there were any posts about issues

**Comment by StoneBlossom** [comment]
Hello! I created this github account just to write a response. As someone who has been in a leadership position, both on discord and in real life, this is not a "power hungry mods" situation. The game is currently run on hardware and server limitations due to being a fan project. There's actually a lot of info in the pins of the discord if you'd have cared to read them before being a jerk to the people running the game you're trying to play lol. Complaining about the "game being unplayable" is not constructive nor an original idea, and maybe having some empathy for the developers would garner you some better reactions. Imagine if you poured your heart and soul into a project only to be constantly told it's not good enough by impatient consumers. Have some appreciation for what is here already! 

tldr: Be nicer smh

**Comment by Jufinda** [comment]
> Hello! I created this github account just to write a response. As someone who has been in a leadership position, both on discord and in real life, this is not a "power hungry mods" situation. The game is currently run on hardware and server limitations due to being a fan project. There's actually a lot of info in the pins of the discord if you'd have cared to read them before being a jerk to the people running the game you're trying to play lol. Complaining about the "game being unplayable" is not constructive nor an original idea, and maybe having some empathy for the developers would garner you some better reactions. Imagine if you poured your heart and soul into a project only to be constantly told it's not good enough by impatient consumers. Have some appreciation for what is here already!
> 
> tldr: Be nicer smh

See that's absolutely fine, I do understand that this is the situation, and I appreciate that the game is fan made etc.

I opened up the game and said to a friend that I would check their discord to see if there was any reports of issues, on the announcement page, there was not, I went to the General chat channel and put the message "the game is literally unplayable" and then with a blink of an eye I was removed from the discord, no one has DMd me to say why, no one has warned me or had a chat with me, nothing, literally nothing, this is why I said it's "power hungry mods" if you're running a game where the moment you say anything even 1% negative results in a ban from the community then the games just going to fail..... it's out me off playing the game as a result, not sure if I want to commit to something where I'm just going to get kicked for next to nothing.

Just for reference, I'm an in game Player Moderator for a popular MMORPG and I always mitigate and then mute if someone continues, I never just pull the trigger the moment I see some one say a negative comment (unless is EXTREMELY offensive)

**Comment by StoneBlossom** [comment]
Wow, I was not expecting a genuine, kind, respectful response. Thank you. After hearing your side a bit more clearly, the situation could have been handled better by the moderators. In the servers I'm a moderator of the common user is relatively young, and I'm used to dealing with people who aren't effective communicators and often say something hurtful to the mod team out of the blue. True, most of it is tiny things that we know not to take personally, but over time when you receive hundreds of those a week, the frustration builds up. I apologize for jumping to the conclusion that this was one of those situations. I'm being extremely genuine when I say I deeply appreciate your time and response. Maintaining a good reputation with your community is important, and I'm sorry the mod team weren't able to keep that for you and the game. 

Rereading my response, I came across as assuming and rude. I'm sorry.

**Comment by Jufinda** [comment]
thats fair enough and i completely understand, luckily im a -cough- couple -cough-  years older than some of these people and enjoy pokemon for the nostalgia.

I didnt take what you said as rude, i understood that my post was originally a bit ignorant, hence the detailed reply, either way, server seem ok today and im playing so it's all good.

no need to apologise :)
